google-cloudevent-types/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>google.events</groupId>
+  <artifactId>google-cloudevent-types</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <name>google-cloudevent-types</name>
+   <url>https://github.com/googleapis/google-cloudevents-java</url>
+  <description>
+    Types in Java for CloudEvents issued by Google.
+  </description>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
```

Comment:
Recommend J8 or higher. J7 is deprecated

Reply:
Changed.